app/instance-initializers/axe-component.js

```diff
@@ -34,21 +34,27 @@ export function initialize(application) {
     /**
      * Turns off the accessibility audit during rendering.
      *
+     * defaults to true in testing, false otherwise
      * @public
      * @type {Boolean}
      */
     turnAuditOff: false,
```

Comment:
Since we're reopening here we could literally just set it the first time on the prototype? `turnAuditOff: !Ember.testing`

This is something that should never change at runtime so that should be safe.